doc/userguide/upgrade.rst

```diff
@@ -82,6 +82,8 @@ Major changes
 - Unknown requirements in the ``requires`` keyword will now be treated
   as unmet requirements, causing the rule to not be loaded. See
   :ref:`keyword_requires`.
+- The following sticky buffers for matching FTP headers have been implemented:
```

Comment:
I don't think we need this here. Did you mix up PRs? https://github.com/OISF/suricata/pull/12432#issuecomment-2602573808

Reply:
This change was intentional in introducing the FTP-related sticky buffers.

We've done this for past releases as well, e.g., the 7 to 8 section mentions the newly added sip related keywords.